### PR TITLE
BSP-333: refactoring

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BSP-333

### Change description ###
Refactoring

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java [hunk 1]
```diff
@@ -72,82 +16,12 @@ public static LocalDate transformFormDateIntoLocalDate(String formFieldName, Str
...
+
     public static final Map<String, String> dischargePeriodicalPaymentSubstituteChecklistToCcdFieldNames =
         new ImmutableMap.Builder<String, String>()
             .put("a lump sum order", "Lump Sum Order")
             .put("a property adjustment order", "Property Adjustment Order")
             .put("a pension sharing order", "Pension Sharing Order")
             .put("a pension compensation sharing order", "Pension Compensation Sharing Order")
             .build();
-        
-    public static final Map<String, String> miamExemptionsChecklistToCcdFieldNames =
```
Comment (Dizrupto) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java]: I am guessing this is not used anymore right?

Comment (yarpo) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java]: yep, this was removed from scope

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/PostcodeValidator.java [hunk 2]
```diff
@@ -1,24 +0,0 @@
-package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.helper;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
-public class PostcodeValidator {
-
-    public static List<String> validatePostcode(Map<String, String> fieldsMap, String postcodeKey) {
```
Comment (Dizrupto) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/PostcodeValidator.java]: Just to confirm, this one was not even used anymore it seems? :)

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java (hunk 1 above)
Comment (yarpo) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java]: and these methods were not called from anywhere.

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/PostcodeValidator.java (hunk 2 above)
Comment (yarpo) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/PostcodeValidator.java]: Yeah, we were using the one already extracted to bsp-common-lib

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java (hunk 1 above)
Comment (jakub-kozlowski) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java]: good point

**Pull request merged**